crates/astria-conductor/src/conductor.rs

```diff
@@ -235,16 +376,62 @@ impl Conductor {
                     info!("reloading is currently not implemented");
                 }
 
-                res = &mut self.sync_done, if !self.sync_done.is_terminated() => {
+                // Start the sequencer reader
+                res = &mut self.da_sync_done, if !self.da_sync_done.is_terminated() && !self.execution_commit_level.is_firm_only() => {
+                    match res {
+                        Ok(()) => {
+                            info!("received sync-complete signal from da reader");
+                        },
+                        Err(e) => {
+                            let error = &e as &(dyn std::error::Error + 'static);
+                            warn!(error, "da sync-complete channel failed prematurely");
+                        }
+                    }
+                    if let Some(sequencer_reader) = self.sequencer_reader.take() {
+
+                        // FIXME: waiting here is a tempory solution. better
+                        // solution would be to not send the da sync complete
+                        // until all tasks for da sync are completed.
+                        // implementing the proper solution would require
+                        // communicating with the executor to make sure that the
+                        // execution queue is empty.
+                        info!("waiting for 10 seconds for async tasks to finish");
+                        time::sleep(Duration::from_secs(10)).await;
```

Comment:
Wouldn't it be ok to just populate the executor queue from the syncer? All new blocks should just be added to the tail?

I'd prefer to address that in this PR.